sentry-kotlin-multiplatform/src/commonMain/kotlin/io/sentry/kotlin/multiplatform/protocol/User.kt

```diff
@@ -2,13 +2,13 @@ package io.sentry.kotlin.multiplatform.protocol
 
 public data class User(
     /** The user's email */
-    var email: String = "",
+    var email: String? = null,
```

Comment:
default values of users should be null, not empty strings

Reply:
It's for convenience in initializing Users since we're also defining the constructor with this in this data class